packages/api-request-builder/src/create-service.js

```diff
@@ -66,6 +67,7 @@ export default function createService(
           ...acc,
           ...query,
           ...queryPage,
+          ...queryLocation,
```

Comment:
Why do you add this here?

Reply:
removed https://github.com/commercetools/nodejs/pull/381/commits/d86d95a6e56544a34f4251ebc8b019391699cc57#diff-feb822a4cb46cfc8410bb7dcb806c4d7R35